canonical/prometheus-pushgateway-k8s-operator
# Ingress and catalogue integration

adds an ingress for the pushgateway UI and a catalogue integration

to test:

```
charmcraft pack
juju deploy cos-lite --trust
juju deploy ./prometheus-pushgateway-k8s_ubuntu-22.04-amd64.charm ppg --resource pushgateway-image=ghcr.io/canonical/prometheus-pushgateway:dev
juju relate traefik ppg
juju relate catalogue ppg
```

navigate to the catalogue, click on it and you should be sent to the ppg's dashboard.

ATM you'll be greeted by a 404, unsure why
